composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "autoload": {
         "psr-4": {
-            "ZF\\OAuth2\\Doctrine\\MutateTableNames\\": "src/"
+            "Laminas\\OAuth2\\Doctrine\\MutateTableNames\\": "src/"
```

Comment:
Please keep `ZF` namespace as it is as by current package name.

Reply:
Reverted